cypress/integration/calendar-page.spec.js

```diff
@@ -52,7 +52,7 @@ describe('Calendar page functions', () => {
  // TODO: check for the rest of the text here
     })
 
-    it('should find the first selectable day', () => {  
+    it.only('should find the first selectable day', () => {  
```

Comment:
This will stop all other tests in this block from running. Is that the intended result?